search/indexing_api.py

```diff
@@ -291,7 +334,7 @@ def create_backing_index():
     return new_backing_index
 
 
-def switch_indices(backing_index):
+def switch_indices(backing_index, object_type):
     """
     Switch the default index to point to the backing index, and delete the reindex alias
```

Comment:
Can you add `object_type` to the docstring?

Reply:
✅